code/game/gamemodes/vampire/vampire.dm

```diff
@@ -219,9 +218,8 @@ You are weak to holy things and starlight. Don't go into space and avoid the Cha
 		/obj/effect/proc_holder/spell/vampire/self/jaunt = 300,
 		/obj/effect/proc_holder/spell/vampire/targetted/enthrall = 300,
 		/datum/vampire_passive/full = 500)
+	var/list/drained_humans = list() //This is a list of people the vampire has drained, and how much blood has been drained from them.
```

Comment:
Leaving hard refs in someone's mind is something that I will absolutely not allow.

Reply:
You can move this to mind.dm, you're in good company there with weird antag shit. Though then you don't track the succ per-vampire, which might not be what you're going for here.

Reply:
You will have to make this use soft refs.